roads.mss

```diff
@@ -756,11 +756,12 @@
 }
 
 
-#roads_high::path_outline_right[zoom>=17][type='path'][can_bicycle='designated'][segregated='yes'],
-#tunnel::path_outline_right[zoom>=17][type='path'][can_bicycle='designated'][segregated='yes'],
-#bridge::path_outline_right[zoom>=17][type='path'][can_bicycle='designated'][segregated='yes'] {
+#roads_high::path_outline_right[zoom>=17][type='path'][can_bicycle='designated'][segregated!='no'],
+#tunnel::path_outline_right[zoom>=17][type='path'][can_bicycle='designated'][segregated!='no'],
+#bridge::path_outline_right[zoom>=17][type='path'][can_bicycle='designated'][segregated!='no'] {
   line-cap: butt;
-  line-color: @path-fill;
```

Comment:
Should have a default color (footway).
Should always has a default value, otherwise this makes more rules, hence more time to generate the xml and to render.